models/contacts.js

```diff
@@ -15,20 +16,93 @@ const listContacts = async () => {
 const getContactById = async (contactId) => {
   try {
     const contacts = JSON.parse(fs.readFileSync(contactsPath));
-    const searchContact = contacts.find((elem) => (elem.id = contactId));
+    const searchContact = contacts.find((elem) => elem.id === contactId);
     console.log(searchContact);
     return searchContact;
   } catch (err) {
     console.log(err.message);
   }
 };
 
-const removeContact = async (contactId) => {};
-
-const addContact = async (body) => {};
-
-const updateContact = async (contactId, body) => {};
-
+const removeContact = async (contactId) => {
+  try {
+    const contacts = await JSON.parse(fs.readFileSync(contactsPath));
+    const contactToRemoveIndex = contacts.findIndex(
+      (elem) => elem.id === contactId
+    );
+    if (contactToRemoveIndex === -1) {
+      return false;
+    } else {
+      const newContactsData = contacts.filter((elem) => elem.id !== contactId);
+      fs.writeFileSync(contactsPath, JSON.stringify(newContactsData));
+      return true;
+    }
+  } catch (err) {
+    console.log(err.message);
+  }
+};
+// w tej funkcji nie miałem początkowo pomysłu w jaki sposób przekazać
+// wynik funkcji z uwagi na wykonania w pętli i uzyłem poprostu liczb
+// które później interpretuję. Czy takie rozwiązanie nie jest zbyt łopatologiczne?
```

Comment:
Takie rozwiązanie może być, ale lepiej byłoby przekazywać stringa z zapisem co to oznacza, np zamiast 4 zwrócić "notUnique"

Reply:
lub tak jak to zrobiłeś niżej przekazywać obiekt message, a potem sprawdzać jednym ifem sprawdzającym czy jest message jeżeli tak to zwracasz tekst błedu a jak nie to zwracasz poprawną wartość